### ISSUE TITLE
Shared Development for cloud customers

### ISSUE THREAD

**Issue by cfarre** [grafana/azure-monitor-datasource#88]
@daniellee 
We at azure solutions and azure cat team have scenarios for enterprise customers at high rank. For example Adobe one of our customers. So there is a requirement for sharing the dashboards at scale. One of the requirements is to be able to replace repetitive configuration. The workspace id is one spot. Provide a way to replace the workspace from a dashboard variable

**Comment by daniellee** [comment]
Yes, should definitely do this. We are in the process of porting this plugin to core Grafana so it will be probably be included in Grafana 6.1 after that work is done. But maybe for 6.0 if we have time.

**Comment by yesoreyeram** [comment]
hi @daniellee Is this coming in 6.1 ?

**Comment by yesoreyeram** [comment]
I just found that I can achive the desired output with the cross application / workspace queries. 

More details available at the below sites

https://docs.microsoft.com/en-us/azure/azure-monitor/log-query/cross-workspace-query
https://dev.applicationinsights.io/documentation/Using-the-API/Cross-Resource-Queries
